src/slash-bedrock/libexec/brl-apply

```diff
@@ -343,6 +343,14 @@ if [ -r /etc/fstab ] && awk '$1$2$3$4$5$6 !~ "#" && $6 != "" && $6 != 0 {x=1} EN
 	' /etc/fstab >/etc/fstab-new && mv /etc/fstab-new /etc/fstab
 fi
 
+# When using 66 init, make a tmpfs available for 66's LIVE directory.
+# The init_cmd variable is exported by bedrock init for use here.
+if [ -n $init_cmd ] && $(echo $init_cmd | grep -q '66'); then
+    live_dir=$(awk -F= -e '/^LIVE=/ { print $2 }' /etc/66/init.conf)
+    mkdir -p $live_dir
+    mount -t tmpfs tmpfs $live_dir -o rw,nosuid,nodev,mode=755,exec
+fi
+
```

Comment:
@paradigm: Ping re. my above questions?

Reply:
Apologies, I'm somewhat overwhelmed with both other Bedrock items that need my attention and non-Bedrock stuff.  I'll see if I can backread here and get back to you within the next week.

Reply:
@flexibeast 

A summary of my understanding of the situation, to make sure we're on the same page:

- Bedrock sets `/run` to have a certain mount property before handing control off to the specified init.
- By default, 66 unmounts (default `-m`) anything found at `/run` (default `LIVE`) and re-mounts over it.  This clears the mount property we want.  This is problem (1) we need to solve.
- By default, Bedrock mounts `/run` with `-onoexec`.  However, per your experimentation 66 apparently requires `/run` (default `LIVE`) to have `-oexec`.  This is problem (2) we need to solve.

To solve (1), we force the removal of `-m`.  As far as I recall, we haven't discussed an alternative.

[To solve (2), we want to respect user settings in `/etc/fstab` vis-a-vis `exec` or `noexec`.  The proper way to do this would be to mount `/etc/fstab` directly in Bedrock's `init` _before_ any other code to mount `/run`.  However, that may be tricky to do at this point in Poki's life without risk of breaking something.  An alternative which may be good enough for this specific effort is to parse `/etc/fstab` with `awk` and remount `/run` with `exec/noexec` in Bedrock's `init`.](https://www.reddit.com/r/bedrocklinux/comments/q55i8r/using_66_in_a_void_stratum/hgh9a7w/)

> When you wrote "the same point where it's doing other /etc/fstab stuff", i take it you mean the "# Configure /etc/fstab" section in `brl-apply`, and not the fstab stuff in `installer.sh`?

I meant Bedrock's `init`, the only place where it mounts `/etc/fstab`.

> assume i'll have to check whether `/run` is mounted `-oexec` by default on Obarun and Artix?

When running 66 without any special patching regarding `-m`, yes, to confirm the theory that 66 requires `LIVE` / `/run` be `-oexec`.

Clearly if it doesn't, then we're barking up the wrong tree with problem (2).

> And what do we do if the user has set LIVE to something outside of `/run`?

Require they update `/etc/fstab` accordingly.